### ISSUE TITLE
unused comparison in PartialEq of FuzzyWindowResult

### ISSUE THREAD

**Issue by boblannon** [mapbox/fuzzy-phrase#59]
Got this warning when compiling on `rustc 1.30.1 (1433507eb 2018-11-07)`, but I think the only relevant thing that has changed since 1.26 is that the compiler now warns about unused comparisons by default.

```
warning: unused comparison which must be used
   --> src/glue/mod.rs:219:9
    |
219 |         self.edit_distance == other.edit_distance;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: #[warn(unused_must_use)] on by default

    Finished dev [unoptimized + debuginfo] target(s) in 21.54s
```

It does look like a legitimate bug in the code.  Presumably these two comparisons were meant to be AND'ed together:

https://github.com/mapbox/fuzzy-phrase/blob/ef94afa475a44543652fa384e6f1cb477395c136/src/glue/mod.rs#L193-L198

**Comment by apendleton** [comment]
Agreed. Good catch.

**Comment by boblannon** [comment]
couldn’t tell at a cursory look but i wonder how often that would end up having real consequences

**Comment by apendleton** [comment]
Probably never. There's not a reason you'd ever compare the results of a window lookup with the results of a regular fuzzy lookup, except in tests (we do that to make sure their results are the same). Still worth fixing though.

**Comment by apendleton** [comment]
This got fixed in #64
